packages/expo/src/environment/logging.fx.ts

```diff
@@ -5,12 +5,19 @@ import RemoteLogging from '../logs/RemoteLogging';
 
 if (Constants.manifest && Constants.manifest.logUrl) {
   // Enable logging to the Expo dev tools only if this JS is not running in a web browser (ex: the
-  // remote debugger)
-  if (!navigator.hasOwnProperty('userAgent')) {
+  // remote debugger). In Expo Web we don't show console logs in the CLI, so there's no special case needed.
+  if (!isRunningInWebbrowser()) {
```

Comment:
```suggestion
  if (!isRunningInWebBrowser()) {
```

This being said, we may want to define `atob` in all Expo environments in the future. I think a safer approach would be to use `Platform.isDOMAvailable` (from `{ Platform } from @unimodules/core`).

Reply:
We probably want to send some config from the CLI instead, for now this approach seems fine.

Reply:
atob is fragile though, that seems like the kind of thing we'd want to define c.f. Deno

Reply:
Unfortunately (for this case), `isDOMAvailable` is `false` in the context of remote debugging, since JS code is executed in a Worker, and `window` is a `DedicatedWorkerGlobalScope` which doesn't expose DOM methods.

We could try to use some other property, like `window.__REACT_DEVTOOLS_GLOBAL_HOOK__` (😕) or instead of using `hasOwnProperty('userAgent')` try to evaluate it (it's `undefined` in the context of mobile device, but a string in the context of the remote JS worker).

Reply:
This is perhaps a bit silly and we'd want to document it in our code, but `navigator.product === 'Gecko'` in all browsers for compatibility reasons: https://developer.mozilla.org/en-US/docs/Web/API/NavigatorID/product

Reply:
That was a very good suggestion — `navigator.product` is `Gecko` in the Worker in which JS code is being run when debugging remotely! In React Native `navigator.product` is `ReactNative`. Changed the PR.